.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,11 @@
+{
+  "ExpandedNodes": [
```

Comment:
Waar is dit voor bedoeld?

Reply:
Deze zijn van me localfiles meegenomen zie ik nu, die heb ik op me huidige versie in de gitignore (als het goed is)